include/dbclean.php

```diff
@@ -18,16 +23,20 @@ function dbclean_run(&$argv, &$argc) {
 		unset($db_host, $db_user, $db_pass, $db_data);
 	}
 
-	load_config('config');
-	load_config('system');
```

Comment:
But please wait for it until the new release is done. @Hypolite had detected some problems with new installations. This (and other bugfixes) should be the only thing that we now accept as a pull request.